Makefile

```diff
@@ -0,0 +1,10 @@
+PROJECT = trails
+
+DEPS = lager sync xref_runner
```

Comment:
`xref_runner` should be in `CT_DEPS`
`lager` shouldn't be required
`sync` should be in `SHELL_DEPS`